src/api/app/controllers/attribute_controller.rb

```diff
@@ -87,54 +87,52 @@ def show_attribute_definition
     status: 404, errorcode: 'unknown_attribute'
   end
 
-  # /attribute/:namespace/:name/_meta
-  def attribute_definition
+  def delete_attribute_definition
     # permission check via User model
     return unless extract_user
 
-    if request.post? || request.put?
-      logger.debug '--- updating attribute type definitions ---'
-
-      xml_element = Xmlhash.parse(request.raw_post)
+    if at = @ans.attrib_types.where('name = ?', name).first
```

Comment:
`@ans.attrib_types.where('name = ?', name).first` can be written like:

`@ans.attrib_types.where(name: name).first` or even better:

`@ans.attrib_types.find_by(name: name)`